src/app/modules/main/chat_section/controller.nim

```diff
@@ -299,8 +299,6 @@ proc init*(self: Controller) =
 
   self.events.on(SIGNAL_CHAT_SWITCH_TO_OR_CREATE_1_1_CHAT) do(e:Args):
     let args = ChatExtArgs(e)
-    if (self.isCommunitySection):
-      return
```

Comment:
This condition is still useful. We don't want communities to react to this, since they don't know the chat section.
Even better, we could move the condition outside like so:
```nim
if (not self.isCommunitySection):
  self.events.on(SIGNAL_CHAT_SWITCH_TO_OR_CREATE_1_1_CHAT) do(e:Args):
    let args = ChatExtArgs(e)
    self.delegate.createOneToOneChat(args.communityId, args.chatId, args.ensName)
```
This way, we only listen if needed